README.md

```diff
@@ -55,6 +55,8 @@ Most developers begin by identifying an interesting sample based on a specific u
   A sample proxy to invoke Cloud Run Service from Apigee
 - [integrated-developer-portal](integrated-developer-portal) -
   This sample demonstrates how to create an Apigee Integrated portal and shows how to expose your API products to its catalog
+- [exposing-to-internet](exposing-to-internet) -
+  This sample demonstrates how to expose an Apigee instance to the internet using an external HTTP(S) load balancer
```

Comment:
can we update `external HTTP(S) load balancer` to [Google Cloud external HTTP(S) Load Balancer](https://cloud.google.com/load-balancing/docs/https)